keyboards/handwired/videowriter/keymaps/via/rules.mk

```diff
@@ -0,0 +1,4 @@
+# this particular convertion project uses Pro Micro with DFU bootloader instead of Caterina
+BOOTLOADER = lufa-dfu
```

Comment:
Usually, this is atmel-dfu, or qmk-dfu.  lufa-dfu is rather uncommon.

Reply:
It's actually a lufa-dfu (from my fork, adapted for Pro Micro, as I use lot of them, not only for QMK). I forgot that this code will be used by everyone, so I'll change the bootloader setting to default Caterina